docs/DownloadBOLs.md

```diff
@@ -0,0 +1,133 @@
+            "driverPickupNotes": null,
+            "pickupDate": "2022-12-01T16:09:41.684+00:00",
+            "dropoffAddress": "200 W Monroe St, Phoenix, AZ 85003, USA",
+            "driverDropoffNotes": null,
+            "dropoffDate": "2022-12-01T16:13:25.965+00:00",
+            "distanceInMiles": 2048,
+            "transportationCompanyAddress": null,
+            "driverFullName": "Bryan Boye",
+            "vehicles": [
+              {
+                "type": "Car",
+                "year": 2012,
+                "make": "Audi",
+                "model": "A7",
+                "isOverSized": false,
+                "isOperational": true,
+                "odometer": 0,
+                "vehicleMeasurements": {
+                  "height": 55.9,
+                  "length": 195.6,
+                  "width": 84.2,
+                  "curbWeight": 4210.0,
+                  "grossVehicleWeightRating": null,
+                  "grossVehicleWeightRangeMin": null,
+                  "grossVehicleWeightRangeMax": null,
+                  "tonnage": null
+                },
+                "vehicleMarket": "LIGHT",
+                "vin": "WAUYGAFC6CN174200",
+                "rolls": true,
+                "missingKeys": false
+              }
+            ]
+          },
+          "pickupSignature": "order-workflow-staging/transportationOrder/08ac661b-105a-4ce5-a4a6-47fe06529847/06021337-7d40-4bc6-9d91-7326d370d713/pickupSignature",
+          "pickupDriversSignature": "order-workflow-staging/transportationOrder/08ac661b-105a-4ce5-a4a6-47fe06529847/06021337-7d40-4bc6-9d91-7326d370d713/pickupDriversSignature",
+          "pickupSignedBy": "John",
+          "pickupSignedAt": {
+            "type": "Point",
+            "coordinates": [null, null]
+          },
+          "dropoffSignature": "order-workflow-staging/transportationOrder/08ac661b-105a-4ce5-a4a6-47fe06529847/06021337-7d40-4bc6-9d91-7326d370d713/dropoffSignature",
+          "dropoffDriversSignature": "order-workflow-staging/transportationOrder/08ac661b-105a-4ce5-a4a6-47fe06529847/06021337-7d40-4bc6-9d91-7326d370d713/dropoffDriversSignature",
+          "dropoffSignedBy": "Jill",
+          "dropoffSignedAt": {
+            "type": "Point",
+            "coordinates": [null, null]
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+To get the RunBuggy PDF of the Bill of Lading, make the following request for each BOL ID and Transportation Order ID retrieved. Replace the items in curl brackets with the respective IDs. Using the example above, the Transportation Order ID would be `08ac661b-105a-4ce5-a4a6-47fe06529847` and the BOL ID would be `06021337-7d40-4bc6-9d91-7326d370d713`.
+
+```plaintext
+curl --location -g --request GET
+'https://apps.runbuggy.com/runbuggy/orders-workflow/transportation-orders/{TRANSPORTATION ORDER ID}/bills-of-lading/{BOL ID}'
+--header 'Authorization: Bearer CHANGEME' \
+--data-raw ''
+```
+
```

Comment:
I'd add a subheader for the following section. It's different than the other part of the guide.